unit_test/sparse/Test_Sparse_spmv.hpp

```diff
@@ -10,6 +12,11 @@
 
 #include "KokkosKernels_Controls.hpp"
 #include "KokkosKernels_default_types.hpp"
+#include "Cuda/Kokkos_Cuda_Half.hpp"
+
+#include "KokkosSparse_BlockCrsMatrix.hpp" 
+#include "KokkosSparse_spmv_tensor_core_impl.hpp"
```

Comment:
We should avoid loading the impl file if possible and rely on the public header to benefit from ETI